src/DateRangePickerInput/web/DateRangePickerInput.js

```diff
@@ -313,6 +317,7 @@ DateRangePickerInput.defaultProps = {
   format: 'YYYY-MM-DD',
   fromMonth: new Date(),
   onDateChange: () => {},
+  disabledDays: [{ before: new Date() }],
```

Comment:
same as above
`Let's not default this to anything. Since the majority usecase is to allow all dates it at all :)`